pyston/order/managers.py

```diff
@@ -219,4 +221,10 @@ class DjangoOrderManager(BaseParserModelOrderManager, BaseDjangoOrderManager):
     """
 
     def _sort_queryset(self, qs, terms):
+        try:
+            if 'id' not in [term.expression.name for term in terms]:
```

Comment:
Nebo me napada pridat promennou do tridy, kterou by tohle slo nastavit. Default bych dal True a vypinalo by se to explicitne.